### PR TITLE
Ethbridge querier and query / msg service stubs

### PR DESCRIPTION
**Pull request by timlind**
@marbar3778 Let me know if things need to be done differently or anything was missed. Would like to get this merged in just focussing on the querier so we're working off a current branch.

### CODE REVIEW

File: proto/sifnode/ethbridge/v1/query.proto [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+option go_package = "github.com/Sifchain/sifnode/x/ethbridge/types";
+
+// Query service for queries
+service Query {
+  // EthProphecy queries an EthProphecy
+  rpc EthProphecy(QueryEthProphecyRequest) returns (QueryEthProphecyResponse) {}
+}
+
+// TODO check rest property names
```
Comment (iverc) [inline proto/sifnode/ethbridge/v1/query.proto:16]: Let's do some cleanup if we merge it in as is so we don't have to do it when it gets bigger

**Review by iverc — APPROVED** [review]

File: x/ethbridge/client/cli/query.go [hunk 2]
```diff
@@ -42,25 +47,21 @@ func GetCmdGetEthBridgeProphecy(queryRoute string, cdc *codec.Codec) *cobra.Comm
...
 
-			var out types.QueryEthProphecyResponse
-			err = cdc.UnmarshalJSON(res, &out)
+			res, err := queryClient.EthProphecy(context.Background(), req)
 			if err != nil {
 				return err
 			}
 
-			return cliCtx.PrintOutput(out)
+			return clientCtx.PrintProto(res)
```
Comment (timlind) [inline x/ethbridge/client/cli/query.go:64]: @marbar3778 what should this line look like?

File: proto/sifnode/ethbridge/v1/query.proto (hunk 1 above)
Comment (marbar3778) [reply proto/sifnode/ethbridge/v1/query.proto:16]: Is this todo carried over from the previous query?

Comment (timlind) [reply proto/sifnode/ethbridge/v1/query.proto:16]: I've added it as there was one field name distinction in the json tag, will update that in the Proto now.

**Pull request merged**